format_test_hits.py

```
@@ -0,0 +1,36 @@
+def format_file(file_path):
+    with open(file_path, "r") as f:
+        lines = f.readlines()
+
+    # Remove the mock import at the end
+    lines = [l for l in lines if not l.startswith("from unittest.mock import patch")]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)